src/status_im/ui/screens/ens/events.cljs

```diff
@@ -0,0 +1,123 @@
+(ns status-im.ui.screens.ens.events
```

Comment:
i believe we don't use events in ui namespace, handlers should be in root events file, fx and functions in core.cljs

Reply:
There are plenty events files.

Reply:
Which is wrote g